### DISCUSSION TITLE
How to program application Image into SPI Flash?

### DISCUSSION THREAD

**Discussion by gege0500** [polarfire-soc/polarfire-soc-documentation#282] [Applications and demos] [answered]
How to program application Image into SPI Flash? Is the image a binary file in bin format?

**Comment by TommyMurphyTM1234** [comment]
Any use?

* https://onlinedocs.microchip.com/pr/GUID-252A769E-4D9A-4D42-965A-A3E3AA43B756-en-US-1/index.html?GUID-508E75A8-BEBD-4E26-995C-2FB88C42E737

**Reply by gege0500** [reply]
Thank you for your reply

**Comment by gege0500** [comment]
收到

**Reply by TommyMurphyTM1234** [reply]
This might also be of interest...

* https://github.com/polarfire-soc/polarfire-soc-documentation/blob/master/boards/mpfs-generic/booting-from-qspi/booting-from-qspi.md

**Reply by gege0500** [reply]
Thank you for your reply

**Comment by hughbreslin** [comment, marked as answer]
Hi @gege0500, have you seen this [document](https://github.com/polarfire-soc/polarfire-soc-documentation/blob/master/boards/mpfs-generic/programming-the-spi-flash-on-a-polarfire-soc-board-and-booting-via-hss/programming-the-spi-flash-on-a-polarfire-soc-board-and-booting-via-hss.md)? It outlines the steps to program an image into the system controller SPI flash.

**Reply by gege0500** [reply]
Thank you for your reply！This file solves my problem！
